src/main/blackbox/blackbox.c

```diff
@@ -345,6 +346,8 @@ static const blackboxSimpleFieldDefinition_t blackboxSlowFields[] = {
     {"wind",                   0, SIGNED,   PREDICT(0),      ENCODING(UNSIGNED_VB)},
     {"wind",                   1, SIGNED,   PREDICT(0),      ENCODING(UNSIGNED_VB)},
     {"wind",                   2, SIGNED,   PREDICT(0),      ENCODING(UNSIGNED_VB)},
+    {"temperature",           -1, SIGNED,   PREDICT(0),      ENCODING(UNSIGNED_VB)},
```

Comment:
perhaps encoding needs to be SIGNED_VB?
FLIGHT_LOG_FIELD_ENCODING_SIGNED_VB       = 0, // Signed variable-byte

Reply:
I was misleaded by the wind elements above, reading further up in the code makes it clear that I should indeed use `SIGNED_VB`.